tools/templates/vfvs_run.py

```diff
@@ -216,32 +222,159 @@ def program_runstring_array(task):
             '--config', task['config_path'],
             '--ligand', task['ligand_path'],
             '--out', task['output_path'],
-            '--log', f"{task['output_path_base']}.flexres.pdb",
-            '--atom_terms', f"{task['output_path_base']}.atomterms"
+            '--log', f"task['output_path_base'].flexres.pdb",
```

Comment:
Should merge in the current code (this is backing out a recent change)

Reply:
Will update!

Reply:
Wanted to confirm: So the --atom_terms line is completely removed?